airbyte-cdk/python/airbyte_cdk/sources/utils/transform.py

```diff
@@ -178,4 +181,10 @@ def transform(self, record: Dict[str, Any], schema: Mapping[str, Any]):
             just calling normalizer.validate() would throw an exception on
             first validation occurences and stop processing rest of schema.
             """
-            logger.warning(e.message)
+            logger.warning(self.get_error_message(e))
+
+    def get_error_message(self, e: ValidationError) -> str:
+        instance_json_type = python_to_json[type(e.instance)]
+        validator_value = str(e.validator_value)
+        key_path = "." + ".".join(e.path)
+        return f"Failed to transform value of type '{instance_json_type}' to '{validator_value}', key path: '{key_path}'"
```

Comment:
Can you also log the actual value in this message? It helps to know what the problem was for debugging